guides/source/configuring.md

```diff
@@ -1047,6 +1047,10 @@ Specifies if an error should be raised if the order of a query is ignored during
 
 Controls whether migrations are numbered with serial integers or with timestamps. The default is `true`, to use timestamps, which are preferred if there are multiple developers working on the same application.
 
+#### `config.active_record.validate_migration_timestamps`
+
+Controls whether to validate migration timestamps. When set, an error will be raised if a timestamp is not a valid timestamp in the form YYYYMMDDHHMMSS. `config.active_record.timestamped_migrations` must be set to `true`. Default is `false`.
```

Comment:
Should we make the default `false` for existing apps, but `true` for new apps? My gut feeling is that in most cases, apps would want to adhere to this, and those that don't want to can simply set it to `false`.

Reply:
That makes sense to me -- making it opt-in was about preventing breakage in existing apps, but does seem like something to enforce for new applications. Do others have opinions on this?

Reply:
I think it's fine to enforce in new apps.

Reply:
Done!